packages/elinks.rb

```diff
@@ -25,6 +25,7 @@ class Elinks < Package
   depends_on 'lzip'
   depends_on 'openssl'
   depends_on 'zlibpkg'
+  depends_on 'libx11'
```

Comment:
Do we actually need libx11?  I'm wondering since this is a console-only app.  If this is actually needed, then we would need to bump the version from `f86be659` to `f86be6597` in order to detect the new changes.  We would also need new binaries.

Reply:
I'm surprised ncurses is not a dependency.  Maybe it should be added?

Reply:
re: libx11... Good point. I'll check out the config and see if x11 can be pulled back, The install without complained of a lack of libx11. I also thought it was odd elinks needs libx11... that's kinda the point of elinks? :)

Reply:
Bumped version to latest; I went back to the original commit and saw that the tarball is coming from a mirrored gitlab... which seems to have broken 2 months ago. I added --without-x to avoid auto-linking to X11 libs.